samples/boards/stm32/pm_lptim/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  name: Power Management LP Timer
+tests:
+  sample.boards.stm32.pm_lptim.lsi:
+    build_only: true
+    depends_on: lptim
```

Comment:
Is it possible to make Power Management on all soc supporting lptim?
Or should we add a dependance on power management ?

Reply:
No, at this step, the LPTIM is used for  counting time (kernel ticks) _only_ in case the soc is activating  the power management